From 6fe4201b6335a494a2bc2077a7bf7a878509f0c7 Mon Sep 17 00:00:00 2001
Message-ID: <6fe4201b6335a494a2bc2077a7bf7a878509f0c7.1770997609.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Thu, 11 May 2023 16:40:03 +0200
Subject: [PATCH] test_pixmap: adjust to turbojpeg

Apparantly, different jpeg libraries decode the test jpg differently.
Adjust to turbojpeg's color count until upstream uses an invariant
format.
---
 tests/test_pixmap.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_pixmap.py b/tests/test_pixmap.py
index 54128e52..1e68b2cc 100644
--- a/tests/test_pixmap.py
+++ b/tests/test_pixmap.py
@@ -105,7 +105,7 @@ def test_color_count():
     e.g. in Linux system installs.
     '''
     pm = pymupdf.Pixmap(imgfile)
-    assert pm.color_count() == 40624
+    assert pm.color_count() == 39912
 
 def test_memoryview():
     pm = pymupdf.Pixmap(imgfile)
-- 
2.53.0.247.g99c804103f

